app/Http/Controllers/Api/TimeLog/ClockInDomain/Services/ClockInService.php

```diff
@@ -1,96 +1,32 @@
 <?php 
 namespace App\Http\Controllers\Api\TimeLog\ClockInDomain\Services;
 
-use function Opis\Closure\serialize;
-
 // Auth
 use Illuminate\Support\Facades\Auth;
-// DomainValue Objects
-use App\DomainValueObjects\UUIDGenerator\UUID;
-use App\DomainValueObjects\TimeLog\ClockIn\ClockIn;
-use App\DomainValueObjects\TimeLog\TimeStamp\TimeStamp;
-// TB Models
-use App\TimeLog;
-use App\TimeSheets;
 
-//Exception
-use App\Exceptions\TimePuncherExceptions\ClockIn\AlreadyClockedIn;
 // Contracts
 use App\Http\Controllers\Api\TimeLog\ClockInDomain\Contracts\ClockInContract;
-use App\Http\Controllers\Api\TimeLog\TimePuncher\Contracts\TimePuncherContract;
+use App\Http\Controllers\Api\TimeLog\Logic\Contracts\ClockInLogicContract;
 
 class ClockInService implements ClockInContract
 {
-    private $domainName = "TimeLog";
-
-    protected $timePuncherRetriever;
-
-    public function __construct(TimePuncherContract $timePuncherContract)
-    {
-        $this->timePuncherRetriever = $timePuncherContract;
-    }
-
-    //TODO: hard code fix
-    private function verifyUserHasNotYetTimeLogged($userId)
-    {
-        $userId = 1;
-        
-        $hasUserTimeLogged = TimeLog::where('user_id', $userId)->where('clock_out',null)->get();
-        
-        if($hasUserTimeLogged->count()!=0 || $hasUserTimeLogged == null){
-            throw new AlreadyClockedIn();
-        }
-        
-        return true;
-    }
-
-    //TODO hard Code fix
-    private function getTimeSheetId($user)
-    {
-        $userId = 1;
-        //add try catch
-        $timeSheet = TimeSheets::where('user_id', $userId)->first();
-        
-        return $timeSheet->id;
-    }
+    protected $clockInLogic;
 
-    private function getTimeLogParam($user,$timeStamp): array
+    public function __construct(ClockInLogicContract $clockInLogic)
     {
-        $logParam = array();
-
-        $logParam['timeSheetId'] = $this->getTimeSheetId($user);
-        
-        $logParam['uuid'] = new UUID($this->domainName);
-
-        $timeStamp = new TimeStamp(new UUID('timeStamp'), $timeStamp);
-        
-        $logParam['clockIn'] = new ClockIn(new UUID('clockIn'), $timeStamp);
-        
-        return $logParam;
+        $this->clockInLogic = $clockInLogic;
```

Comment:
variables referring to contracts should be consistent throughout the app. We probably need to decide on a universal term for this